config.go

```diff
@@ -19,14 +19,12 @@ import "fmt"
 //go:generate paramgen -output=paramgen_dest.go DestinationConfig
 
 type DestinationConfig struct {
-	// The keyspace (similar to a database in a relational database system) that has the table.
+	// The keyspace name that has the table (similar to a database in a relational database system).
 	Keyspace string `json:"keyspace" validate:"required"`
 	// The table name.
 	Table string `json:"table" validate:"required"`
-	// The host to access Cassandra.
-	Host string `json:"host" validate:"required"`
-	// Cassandra’s TCP port.
-	Port int `json:"port" default:"9042"`
+	// Comma separated list of Cassandra nodes' addresses (at least one), ex: 127.0.0.1:9042,127.0.0.2:8080
+	Nodes []string `json:"nodes" validate:"required"`
```

Comment:
@maha-hajja sorry, I'm a bit late to this, but it would be nice if we would validate that the supplied nodes are valid.

Reply:
@lovromazgon nw! you mean format wise?

Reply:
Yeah, otherwise we would allow a user to specify nodes that are clearly invalid and only find out when they try to run the pipeline.

Reply:
one this is that the supplied node can be only the host without the port if the port is 9042 (cuz it's the default port), so I can validate that it's one of these (a host format, or host+port format)
I didn't want to allow this at first because I didn't want the description to be too long and give the user too many options, you can even use `localhost` instead of `127.0.0.1`, should i keep all these options valid? wdyt?

Reply:
The description is fine, I wouldn't change it. `localhost` should be allowed, yes. I'd simply validate that the node addresses are valid hostnames (or hostname:port pairs), similar to [`hostname`](https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Hostname_RFC_952) or [`hostname_port`](https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-HostPort) validations in the playground validator.